src/lib/plugin.js

```diff
@@ -127,6 +127,12 @@ class FiveBellsLedger extends EventEmitter2 {
     const ledgerMetadata = yield this._fetchLedgerMetadata()
     this.accountUriTemplate = ledgerMetadata.urls.account
 
+    // Set ILP prefix
+    this.prefix = ledgerMetadata.ilp_prefix || this.fallbackPrefix
+    if (!this.prefix) {
+      throw new Error('Unable to set prefix from ledger or from local config')
+    }
```

Comment:
It seems like maybe it should throw an error if the prefix you supply doesn't match what the ledger says. Is there any good reason why this might happen? If there were I would think that the configured prefix should actually override the one from the ledger, rather than the other way around.